include/utils.h

```diff
@@ -174,7 +162,9 @@ typedef void(DirFunc)(void*);
 int processdirs(DirFunc dir_fn);
```

Comment:
cleanup

Reply:
I think you removed the need for this, right?

Reply:
bfmi was not updated and still requires the old functions for C-Thread-Pool.

Reply:
50687c1